dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/model/FromInstructionTest.java

```diff
@@ -239,4 +239,23 @@ public void testInvalidFromData(String input) {
     public void isFromInstructionWithThisImageAndOlderTag(String line, String imageName, String imageTag, boolean expectedResult) {
         assertEquals(FromInstruction.isFromInstructionWithThisImageAndOlderTag(line, imageName, imageTag), expectedResult);
     }
+
+    @DataProvider
+    public Object[][] commentDataNoDfiu() {
+        return new Object[][] {
+                {"FROM image:tag as builder",       true},
+                {"FROM image:tag#no-dfiu",          false},
+                {"FROM image:tag # no-dfiu",        false},
+                {"FROM image:tag\t# no-dfiu",       false},
+                {"FROM image:\t# no-dfiu # # # ",   false},
+                {"FROM image:",                     true},
+                {"FROM image:test # :no-dfiu",      false},
+                {"FROM image:test # no-dfiu added comments for ignoring dfiu PR",      false}
```

Comment:
Can you add a few more: “FROM image:no-dfiu” and “FROM no-dfiu”

Reply:
Done